os/net/netdev/netdev_ioctl.c

```diff
@@ -1218,6 +1259,40 @@ int lwip_func_ioctl(int cmd, void *arg)
 		in_arg->req_res = ret;
 		ret = 0;
 		break;
+#endif
+#if defined(CONFIG_NET_LWIP_DHCP)
+	case DHCPCSTART:
```

Comment:
Here is end node of ioctl, you should return one of error in errno.h
Then vfs will set it as errno...

Reply:
You are right. Applied. Thanks.